src/components/About/Team/index.js

```diff
@@ -133,26 +133,25 @@ const Description = styled.div`
 
 const Team = props => {
   let [contributors, updataContributors] = useState([]);
-  /*eslint-disable */
 
-  useEffect(async () => {
-    // Adding title tag to page
+  useEffect(() => {
     document.title =
       'Developer Team of SUSI.AI - Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots';
 
-    let data = await getContributors();
-    let flags = [],
-      output = [];
-    for (let i = 0; i < data.length; i++) {
-      if (flags[data[i].name]) {
-        continue;
+    (async function fetchData() {
```

Comment:
Why to name the function, `fetchData` is a very common function name and can be used for future.

Reply:
@anishagg17 please review I have changed the function the name.